README.md

```diff
@@ -225,21 +225,21 @@ This process will be called every time a Looker embed IFRAME is created. The acq
 
 Cookieless embed sessions are associated with the user's browser user agent. It is important that that user agent for the browser be set on the request.
 
-The `acquire_embed_cookieless_session` returns a number of tokens if successful:
+If successful, the `acquire_embed_cookieless_session` returns a number of tokens:
 
-- `session_reference_token` - this token is used to generate new tokens. If it exists, it is used to attach to an existing session. It is important to secure this token and it should not be exposed to the browser. This token lives for the duration of the session. Note that a new cookieless embed session will need to be created if the `session_reference_token` has expired.
+- `session_reference_token` - this token is used to generate new tokens and created new IFRAMEs. It is important to secure and keep track of this token. It should not be returned to the browser. This token lives for the duration of the session. A new cookieless embed session will need to be created when the `session_reference_token` expires.
 - `authentication_token` - this is one time token that has a lifespan of 30 seconds. It is used with the `/login/embed/{target}` endpoint.
 - `navigation_token` - this token is used to navigate to different Looker pages in the Looker application. This token lives for 10 minutes.
 - `api_token` - this token is used for api calls. This token lives for 10 minutes.
 
-A time to live for each token is also returned for each token. It is important that the response of the `acquire_embed_cookieless_session` be returned to the browser with the exception of the `session_reference_token`.
+A time to live for each token is also returned. It is important that the response of the `acquire_embed_cookieless_session` be returned to the browser with the exception of the `session_reference_token`. The hosting application MUST keep track of the `session_reference_token` for each user.
 
-The example below uses javascript, but you may utilize the Looker SDK in the language of your choice. Note that this is very simplistic implementation for demonstration purposes only. An actual implementation should be a lot more robust.
+The example shown below is simplistic and uses an in memory cache to keep track of the `session_reference_token`. In memory caches will not work in clustered environments so a use a distributed cache such as `redis` in production. An alternative is to save the `session_reference_token` in an encrypted session cookie. The use of session cookies is demonstrated [here](/server_utils/routes.ts).
```

Comment:
```suggestion
The example shown below is simplistic and uses an in memory cache to keep track of the `session_reference_token`. In memory caches will not work in clustered environments so use a distributed cache such as `redis` in production. An alternative is to save the `session_reference_token` in an encrypted session cookie. The use of session cookies is demonstrated [here](/server_utils/routes.ts).
```

Reply:
I'm including this change in the next PR (against this branch) coming up soon.